boost-common/src/main/java/io/openliberty/boost/common/config/BoosterPackConfigurator.java

```diff
@@ -25,6 +25,7 @@
 
     protected String EE_7_VERSION = "0.1-SNAPSHOT";
     protected String EE_8_VERSION = "0.2-SNAPSHOT";
+    protected String MP_20_VERSION = "0.2-SNAPSHOT";
```

Comment:
So what happens when we add MP 2.1?   What about MP_20 => "0.20-SNAPSHOT"   and MP_21 => "0.21...." ?